cmd/run.go

```diff
@@ -407,13 +386,48 @@ a commandline interface for interacting with it.`,
 			<-sigC
 		}
 
+		reportWg.Wait()
```

Comment:
I wonder if it won't be better if instead of having http timeout, we wait here for n seconds (let say 3 again) and if it doesn't end by then just to continue executing ....(this can be done with a simple `select` and `chan struct{}` that get closed by the report goroutine)
This will both mean that any bugs in how timeout works will be circumvented (although I just read the documentation and I think it takes into account all the things).
It also means that if the request would take 5 seconds, but also the summary will take 6, the report will actually finish ...

Reply:
Sounds good, let me know if 963ca44c does what you had in mind.

Reply:
mostly - yes, but I didn't want `reportUsage` to know about the channel, it can just be closed after the call to `reportUsage` :D

Reply:
also I think this will make the user wait for 3 seconds *always* if they have disabled the report

Reply:
D'oh, you're right about that last point. Please hold :)

Reply:
OK, hopefully 3a89667b resolves all issues.